### PR TITLE
Bumped 1-21 version to 1.21.9 and updated patches

### PR DESCRIPTION
**Pull request by kschumy**
*Description of changes:*
* Bumped Kubernetes version to 1.21.9
* Updated patches
  * ~~⚠️ It's kinda hard to see, but there were actual changes to `0021-EKS-PATCH-Parse-ipv6-address-before-comparison-10773.patch`~~

----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
I'll update checksums

**Comment by kschumy** [comment]
I think the test failures are due to the changes to 021-EKS-PATCH-Parse-ipv6-address-before-comparison-10773.patch. This wasn't a change I felt great about. I'm going to revert the test in setters_test.go

File: projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch [hunk 1]
```diff
@@ -1,43 +0,0 @@
-From 74feb0759484202496e7639c5645e49a7258c9d5 Mon Sep 17 00:00:00 2001
```
Comment (jaxesn) [inline projects/kubernetes/kubernetes/1-21/patches/0010-PATCH-kubeadm-CoreDNS-permissions-for-endpointslices.patch:1]: same question from 1.20, i think this is still needed?

**Review by jaxesn — CHANGES_REQUESTED** [review]
Were all of these removed patches merged upstream in this 1.21 verison?

**Comment by kschumy** [comment]
The CVE patch is no longer needed. This [upstream PR](https://github.com/kubernetes/kubernetes/pull/106961) added it to 1.21.  This [commit](https://github.com/kubernetes/kubernetes/commits/v1.21.9?after=b631974d68ac5045e076c86a5c66fba6f128dc72+51&branch=v1.21.9) is in the 1.21.9 branch before the release.

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**